subprojects/testing-native/src/main/java/org/gradle/nativeplatform/test/xctest/tasks/XcTest.java

```diff
@@ -40,7 +42,7 @@
     public XcTest(ObjectFactory objectFactory) {
         setTestExecuter(objectFactory.newInstance(NativeTestExecuter.class));
 
-        getExtensions().getExtraProperties().set("testBundleDir", newInputDirectory());
+        getExtensions().getExtraProperties().set("testBundleDir", newInputFile());
```

Comment:
Why are these being added as extra properties vs just regular fields?

Reply:
You are right, we can address this once we tackle the test infrastructure cleanup between Java and native.